ci-operator/step-registry/ipi/azure/pre/resourcegroup/ipi-azure-pre-resourcegroup-chain.yaml

```
@@ -0,0 +1,8 @@
+chain:
+  as: ipi-azure-pre-resourcegroup
+  steps:
+  - chain: ipi-conf-azure-resourcegroup
+  - chain: ipi-install
+  documentation: |-
+    The IPI setup step contains all steps that provision an OpenShift cluster
+    in an existing Resource Group configuration on Azure.
```

I don't think we want a new line here. 
```suggestion
    The IPI setup step contains all steps that provision an OpenShift cluster in an existing Resource Group configuration on Azure.
```